### PR TITLE
chore: Add `explorer_create_env` makefile target and update error messages.

### LINKED ISSUE #966: Explorer: If you don't have the local env, it suggests running make create-env which doesn-t exist
If you don't have the local env, it suggests running make create-env which doesn't exist, we should have a local env

### PR DESCRIPTION
**Pull request by JulianVentura**
### Description

* Added new makefile target `explorer_create_env` which creates a new `.env` based on `.env.dev`
* Updated error messages to reflect changes

Closes #966

### CODE REVIEW

File: explorer/lib/explorer/contract_managers/aligned_layer_service_manager.ex [hunk 1]
```diff
@@ -26,7 +26,7 @@ defmodule AlignedLayerServiceManager do
 
     :error ->
       raise(
-        "Config file not read successfully, did you run make create-env? If you did,\n make sure Alignedlayer config file is correctly stored"
+        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure Eigenlayer config file is correctly stored"
```
Comment (uri-99) [inline explorer/lib/explorer/contract_managers/aligned_layer_service_manager.ex]: No, here we are reading the AlignedLayer config file, reading its deployed addresses, not Eigen's.
```suggestion
        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure AlignedLayer config file is correctly stored"
```

File: explorer/lib/explorer/contract_managers/batcher_payment_service_manager.ex [hunk 2]
```diff
@@ -25,7 +25,7 @@ defmodule BatcherPaymentServiceManager do
 
     :error ->
       raise(
-        "Config file not read successfully, did you run make create-env? If you did,\n make sure Alignedlayer config file is correctly stored"
+        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure Eigenlayer config file is correctly stored"
```
Comment (uri-99) [inline explorer/lib/explorer/contract_managers/batcher_payment_service_manager.ex]: Same here
```suggestion
        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure AlignedLayer config file is correctly stored"
```

File: explorer/lib/explorer/contract_managers/avs_directory_manager.ex [hunk 3]
```diff
@@ -14,7 +14,7 @@ defmodule AVSDirectoryManager do
 
     :error ->
       raise(
-        "Config file not read successfully, did you run make create-env? If you did,\n make sure Eigenlayer config file is correctly stored"
+        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure Eigenlayer config file is correctly stored"
```
Comment (uri-99) [inline explorer/lib/explorer/contract_managers/avs_directory_manager.ex:17]: Here it is OK because avs_directory is an Eigenlayer contract

**Review by uri-99 — CHANGES_REQUESTED** [review]

File: explorer/lib/explorer/contract_managers/aligned_layer_service_manager.ex (hunk 1 above)
Comment (JulianVentura) [reply explorer/lib/explorer/contract_managers/aligned_layer_service_manager.ex]: Done

File: explorer/lib/explorer/contract_managers/batcher_payment_service_manager.ex (hunk 2 above)
Comment (JulianVentura) [reply explorer/lib/explorer/contract_managers/batcher_payment_service_manager.ex]: Done

File: explorer/lib/explorer/contract_managers/avs_directory_manager.ex (hunk 3 above)
Comment (JulianVentura) [reply explorer/lib/explorer/contract_managers/avs_directory_manager.ex:17]: Ok, ty for pointing that out!

**Review by uri-99 — APPROVED** [review]

**Review by Oppen — APPROVED** [review]

**Pull request merged**